pkg/handler/collector/deps_dev/deps_dev.go

```diff
@@ -229,15 +237,15 @@ func (d *depsCollector) fetchDependencies(ctx context.Context, purl string, docC
 	return nil
 }
 
-func (d *depsCollector) collectAdditionalMetadata(ctx context.Context, system, name, version string, pkgComponent *PackageComponent) error {
+func (d *depsCollector) collectAdditionalMetadata(ctx context.Context, system, namespace, name, version string, pkgComponent *PackageComponent) error {
 	sys, err := parseSystem(system)
 	if err != nil {
 		return err
 	}
 	versionReq := &pb.GetVersionRequest{
 		VersionKey: &pb.VersionKey{
 			System:  sys,
-			Name:    name,
+			Name:    namespace + "/" + name,
```

Comment:
nit may be more robust with trim postfix in case there is already a "/" in namespace (shouldn't be though)